crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -4203,34 +4130,10 @@ impl AuthorityPerEpochStore {
         &self,
         last: Option<CheckpointHeight>,
     ) -> SuiResult<Vec<(CheckpointHeight, PendingCheckpointV2)>> {
-        let db_results = if !self
-            .epoch_start_config()
-            .is_data_quarantine_active_from_beginning_of_epoch()
-        {
-            // Reading from the db table is only need when upgrading to data quarantining
-            // for the first time.
-            let tables = self.tables()?;
-            let db_iter = tables
-                .pending_checkpoints_v2
```

Comment:
The table pending_checkpoints_v2 is marked as deprecated but is still referenced here. There is no writer for the table, so it should always be empty.
Just double-checking if it's safe to remove this code

Reply:
cc @mystenmark

Reply:
yeah we don't store pending checkpoints in the db anymore so this should be okay